tokio-console/src/view/styles.rs

```diff
@@ -127,6 +127,25 @@ impl Styles {
         )
     }
 
+    pub fn selected<'a>(&self, value: &str) -> Span<'static> {
+        let style = if let Some(cyan) = self.color(Color::Cyan) {
+            Style::default().fg(cyan)
+        } else {
+            Style::default().remove_modifier(Modifier::REVERSED)
+        };
+        Span::styled(value.to_string(), style)
+    }
+
+    pub fn ascending(&self, value: &str) -> Span<'static> {
+        let value = format!("{}{}", value, self.if_utf8("▵", "+"));
+        self.selected(&value)
+    }
+
+    pub fn descending(&self, value: &str) -> Span<'static> {
+        let value = format!("{}{}", value, self.if_utf8("▿", "-"));
```

Comment:
i think it might look a little bit nicer if we added a space after the symbol in the case where it's a plus or minus sign, but not a blocker.